### PR TITLE
CVSL-1599 Adding missing double click protection

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/views/pages/vary-approve/view.njk [hunk 1]
```diff
@@ -30,7 +30,8 @@
                 <div class="govuk-button-group">
                     {{ govukButton({
                         text: "Approve variation",
-                        attributes: { 'data-qa': 'approve-variation' }
+                        attributes: { 'data-qa': 'approve-variation' },
+                        preventDoubleClick: true
                     }) }}
```
Comment (andrewrlee) [inline server/views/pages/vary-approve/view.njk:35]: multiple clicks here lead to duplicate emails being sent and licence events

File: server/views/pages/vary/variationSummary.njk [hunk 2]
```diff
@@ -27,7 +27,8 @@
                 <div class="govuk-button-group">
                     {{ govukButton({
                         text: "Send for approval",
-                        attributes: { 'data-qa': 'send-for-approval' }
```
Comment (andrewrlee) [inline server/views/pages/vary/variationSummary.njk:30]: multiple clicks here lead to duplicate emails being sent and licence events

File: server/views/partials/varyCallToActions.njk [hunk 3]
```diff
@@ -16,7 +16,8 @@
                     text: "Set licence to active",
                     classes: "govuk-button--primary govuk-!-margin-right-1",
                     type: "submit",
-                    attributes: { 'data-qa': 'view-and-print' }
+                    attributes: { 'data-qa': 'view-and-print' },
+                    preventDoubleClick: true
```
Comment (andrewrlee) [inline server/views/partials/varyCallToActions.njk:20]: multiple clicks triggers creation of duplicate events on old and new licences

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**